README.md

```diff
@@ -61,6 +68,9 @@ Please see the Examples section for more details.
 
 ## Versions
 
+### Unreleased
+* Decoupled VM generation from underlying VHD formar in xVMHyperV resourse.
```

Comment:
Thanks - Fixed :). I've also added some more information surrounding the __breaking change__.